jbi/runner.py

```diff
@@ -27,8 +27,8 @@ def execute_action(
     The value returned by the action call is returned.
     """
     log_context = RunnerLogContext(
-        bug=request.bug.dict(),
-        request=request.dict(),
+        bug=request.bug,
+        request=request,
```

Comment:
drive by 🙄

Reply:
Accident?

Reply:
😄  Somewhere between #216 and #222/#219 since the PRs were done simultaneously